### PR TITLE
Refactoring du code

### PR DESCRIPTION
**Pull request by daniel3322612**

### CODE REVIEW

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import com.celestabank.celestabankapi.service.AccountServiceImp;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/account")
+@AllArgsConstructor
```
Comment (daniel3322612) [inline src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:20]: Use @RequiredArgs

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+public class AccountController {
+
+    AccountServiceImp   accountServiceImp;
```
Comment (daniel3322612) [inline src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:23]: Should be private final

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    @PostMapping("/add")
+    public List<Account> addAccount(@RequestBody Account account){
+        return accountServiceImp.addAccount(account);
+    }
+
+    @PostMapping("/savings")
+    public List<SavingAccount> addSavingAcc(@RequestBody SavingAccount savingAccount) throws InvalidDetailsException {
```
Comment (daniel3322612) [inline src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:31]: Spring recommends to use RuntimeExeptions

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java (hunk 1 above)
Comment (samuelpiedjou237) [reply src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:20]: okay sir

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java (hunk 2 above)
Comment (samuelpiedjou237) [reply src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:23]: ok

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java (hunk 3 above)
Comment (samuelpiedjou237) [reply src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:31]: i have some problems to throws my exception to my client

File: src/main/java/com/celestabank/celestabankapi/controller/AccountController.java (hunk 1 above)
Comment (samuelpiedjou237) [reply src/main/java/com/celestabank/celestabankapi/controller/AccountController.java:20]: how can i make this

**Pull request closed without merging**